notification/src/main/java/com/chronica/notification/controller/NotificationController.java

```diff
@@ -0,0 +1,44 @@
+package com.chronica.notification.controller;
+
+
+import com.chronica.notification.data.dto.request.CreateNoticeRequestDto;
+import com.chronica.notification.data.dto.request.QueryNoticeRequestDto;
+import com.chronica.notification.data.dto.request.UpdateNoticeRequestDto;
+import com.chronica.notification.data.dto.response.*;
+import com.chronica.notification.logic.NotificationApi;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(path = "/api/notifications", produces = {MediaType.APPLICATION_JSON_VALUE})
+@RequiredArgsConstructor
+public class NotificationController {
+    private final NotificationApi notificationApi;
+
+    @PostMapping(path = "/create")
+    public ResponseEntity<CreateNoticeResponseDto> createNotice(@RequestBody CreateNoticeRequestDto request){
+        return notificationApi.createNotice(request);
+    }
+
+    @GetMapping(path = "/read/{id}")
+    public ResponseEntity<ReadNoticeResponseDto> readNotice(@PathVariable Long id){
+        return notificationApi.readNotice(id);
+    }
+
+    @PostMapping(path = "/query-all")
```

Comment:
Z tego co rozumiem, to chodzi tutaj o getAll? Jeśli tak to powinien być get, bez żadnej ściezki